nanowallet/Jenkinsfile

```diff
@@ -0,0 +1,6 @@
+defaultCiPipeline {
+	platform = ['ubuntu']
+	ciBuildDockerfile = 'node10.Dockerfile'
+
+	packageId = 'nem-nanaowallet'
+}
```

Comment:
no linter ?

Reply:
I tried to add linting and hitting a log of errors(~3K).  most seems to be valid so will need through them.
This is an example.
```
/miscell/nanowallet/src/app/modules/voting/polls/polls.controller.js
  169:9   error  'Promise' is not defined       no-undef
  171:18  error  'd' is defined but never used  no-unused-vars
  193:19  error  'e' is defined but never used  no-unused-vars
  223:19  error  'e' is defined but never used  no-unused-vars
  327:26  error  'e' is defined but never used  no-unused-vars
  351:19  error  'e' is defined but never used  no-unused-vars
  370:23  error  'e' is defined but never used  no-unused-vars
```